munch-match/src/components/SlotMachine/SlotMachine.jsx

```diff
@@ -26,50 +41,76 @@ const SlotMachine = () => {
     { name: 'chicken', image: chicken },
     { name: 'dessert', image: dessert },
     { name: 'fruit', image: fruit },
-    { name: 'pizza', image: pizza },
+      { name: 'pizza', image: pizza },
     { name: 'sushi', image: sushi },
```

Comment:
Could these not be hardcoded, we can create a loop and use the json we created in assignment 2 during refactoring.